src/contracts/signer-identity.spec.ts

```diff
@@ -364,4 +378,221 @@ describe('signer-identity (identity based signer)', function() {
       });
     });
   });
+
+  describe('when dealing with encryption for identity based accounts', async () => {
+    let cryptoProvider: CryptoProvider;
```

Comment:
I guess that most of the `let` can also be `const` ?

Reply:
Must be let for every variable, because they are set within the before section and used within each test.